src/components/CustomButtons/CustomDetailButton/CustomDetailButton.tsx

```diff
@@ -22,8 +20,8 @@ const StyledCustomDetailButton = styled.button`
   }
 `;
 
-export const CustomDetailButton = ({ children }: { children: React.ReactNode }) => (
-  <ThemeProvider theme={CustomColors}>
-    <StyledCustomDetailButton>{children}</StyledCustomDetailButton>
-  </ThemeProvider>
-);
+export const CustomDetailButton = ({
```

Comment:
existence této komponenty mi příjde úplně zbytečná, podle mě komponenta nepřidává žádnou hodnotu pro `StyledCustomDetailButton`